extensions/wikia/WikiaMaps/js/WikiaMapsCreateMapPreview.js

```diff
@@ -237,8 +235,7 @@ define(
 		function trackMapCreation(tileSetData) {
 			var tileSetId = tileSetData.tileSetId,
 				mapTypeChosen = tileSetData.type,
-				label = mapTypeChosen + '-map-created' +
-					((!tileSetId && mapTypeChosen !== 'geo') ? '-with-new-tileset' : '');
+				label = mapTypeChosen + '-map-created' + (!tileSetId ? '-with-new-tileset' : '');
```

Comment:
Good point! I've been thinking about it while removing code and decided to leave it, so we can check data in GA after releasing it and then clean this part. Also, once we remove it we'll remove the sign for others of the possibility to create strict types of maps.

But once I wrote my thoughts I think those are not strong points ;) Let me try and remove `tileSetData.type` from here ;)